app/models/katello/host_collection.rb

```diff
@@ -107,7 +108,7 @@ def refreshed_jobs
     end
 
     def consumer_ids
-      self.systems.pluck(:uuid)
+      self.hosts.pluck(:uuid)
```

Comment:
hosts don't have UUID's, but their content_facet does
you'd also need to handle cases where there is no content_facet or the uuid of the facet is nil